Colours_And_Fonts_style_v1.css

```diff
@@ -50,7 +115,7 @@ p {
 
 .header {
   box-sizing: border-box;
-  background-color: #2c7da0;
+  background-color: #3750A5;

```

Comment:
I changed the colour of the header from a dark navy blue to a light navy blue that relates to the colours of the Tawa College colours